### DISCUSSION TITLE
Is it possible to have extra-pages that don't appear in the top menu?

### DISCUSSION THREAD

**Discussion by andrewpatto** [CSCfi/rems#2982] [Ideas]
We have some datasets that we'd like to share via REMS - but the datasets themselves don't really have canonical 'home' pages we can point people to for information about the goals of the studies / cohort info etc.

We were thinking it might be easy to have an 'extra-page' for each dataset - all in the one spot in REMS - and just link to from the 'About' page (or elsewhere).

However - whilst I can add extra pages that appear explicitly named as part of the top menu - I can't work out how to add extra pages that are not in the menu bar? Is this possible already and I've missed how to config it?

**Comment by Macroz** [comment]
Hi,

Unfortunately, at the moment it's not possible. The original idea was to have more items in the menu, so that would be contrary to that idea to not have them there. But I see no reason why it couldn't be made to work. 

I remember we have had the need to customize the links in the footer too. To be able to customize some extra pages (or just links) to appear both in the header and the footer could be nice. If the requirements wrt. styling each are not great, I'm sure it's not much work at all. And in your case it's not much work for us to add this, if it seems like useful to you.

**Reply by Macroz** [reply]
I created the issue #2983 from this discussion. Feel free to comment there if you have specific feature requests. We'll discuss this issue.

**Reply by Macroz** [reply]
PR https://github.com/CSCfi/rems/pull/3070 implements this feature (and a bunch of other improvements and consistency). Please have a look. If you have comments or other ideas then let's talk about them. This will likely appear in the next release once it gets merged first.

**Reply by andrewpatto** [reply]
Thanks - will get a chance to try this out next week

**Comment by andrewpatto** [comment]
I have deployed this to our test instance and confirmed this feature works as expected - it hits all our requirements. Thanks

**Comment by Macroz** [comment]
This was released recently in [v2.31](https://github.com/CSCfi/rems/discussions/3091).
